packages/form-helpers/tests/submit.test.ts

```diff
@@ -63,7 +63,7 @@ describe('The submit form helper', () => {
   });
 
   it('will emit an event that bubbles', async () => {
-    const onSubmit = (event: SubmitEvent) => {
+    const onSubmit = (event: Event) => {
```

Comment:
Can this still be a `SubmitEvent` with the polyfill approach? It's a test, so no need to worry to much, but the more normalized the better, right?

Reply:
the only issue is the missing `submitter` property that SubmitEvent has that Event doesnt...I'll def change the type tho